src/patch_parse.c

```diff
@@ -328,7 +328,8 @@ static int parse_header_start(git_patch_parsed *patch, git_patch_parse_ctx *ctx)
 	 * proceeed here. We then hope for the "---" and "+++" lines to fix that
 	 * for us.
 	 */
-	if (!git_parse_ctx_contains(&ctx->parse_ctx, "\n", 1)) {
+	if (!git_parse_ctx_contains(&ctx->parse_ctx, "\n", 1)
+			&& !git_parse_ctx_contains(&ctx->parse_ctx, "\r\n", 2)) {
```

Comment:
Style nit:

```
if (!git_parse_ctx_contains(&ctx->parse_ctx, "\n", 1) &&
    !git_parse_ctx_contains(&ctx->parse_ctx, "\r\n", 2))
```

Continuation lines are aligned with the opening brace and the boolean operators belong to the previous line.

Reply:
Should I align with a mix of tabs and spaces?

Reply:
And how wide are the tabs? 8 cols?